README.md

```diff
@@ -1,36 +1,119 @@
 
-In addition to this one, we use many other repositories in the project.
+  Documentation: [Read more](https://github.com/singnet/das-serverless-functions)
 
-* [Query Engine](https://github.com/singnet/das-query-engine) - Source code for [hyperon-das](https://pypi.org/project/hyperon-das/), the Python library with the main [DAS API](https://singnet.github.io/das-query-engine/api/DAS/).
-* [Atom DB](https://github.com/singnet/das-atom-db) - Source code for [hyperon-das-atomdb](https://pypi.org/project/hyperon-das-atomdb/), a Python library required by hyperon-das to interface with DBMSs and other data storage structures.
-* [Serverless Functions](https://github.com/singnet/das-serverless-functions) and [OpenFaaS Templates](https://github.com/singnet/das-openfaas-templates) - Source code of client/server communication components in DAS Server FaaS architecture.
-* [Attention Broker](https://github.com/singnet/das-attention-broker) - DAS component which keeps track of atom's importance values attached to different contexts and update those values according to the queries made by users using context specific hebbian networks.
-* [DAS MeTTa Parser](https://github.com/singnet/das-metta-parser) - A simplified MeTTa parser used to feed knowledge bases into DAS.
-* [DAS Server Toolbox](https://github.com/singnet/das-toolbox) - A set of tools to deploy and setup a DAS server.
-* Infrastructure setup - Tools and assets required to provision and setup servers in cloud providers.
-    * [Server provisioning in Vultr](https://github.com/singnet/das-pre-infra-vultr)
-    * [Server setup Vultr](https://github.com/singnet/das-infra-stack-vultr)
-    * [Server provisioning in AWS](https://github.com/singnet/das-pre-infra-aws)
-    * [Server setup AWS](https://github.com/singnet/das-infra-stack-aws)
+- **DAS MeTTa Parser**
+  A simplified MeTTa parser used to feed knowledge bases into DAS.  
+
+  Documentation: [Read more](https://github.com/singnet/das-metta-parser)
+
+- **DAS Server Toolbox**
+
+  This CLI provides a set of commands to manage containerized services, OpenFaaS functions, and Metta operations. It includes functionalities to start and stop db services, manage OpenFaaS functions, load Metta files, and validate Metta file syntax.
+
+  Documentation: [Read more](https://github.com/singnet/das-toolbox)
+
+- **Infrastructure Setup**
+
+  Tools for provisioning and setting up servers on cloud providers.  
+
+  Documentation:  
+  - [Server provisioning in Vultr](infrastructure/vultr/provisioning/README.md)  
+  - [Server setup Vultr](infrastructure/vultr/setup/README.md)  
+  - [Server provisioning in AWS](infrastructure/aws/provisioning/README.md)  
+  - [Server setup AWS](infrastructure/aws/setup/README.md)
+
+## **Getting Started**
+
+### **1. Cloning the Repository**
+```bash
+git clone <repo-url>
```

Comment:
Please put the actual URL here so readers can just copy-and-paste in their terminals.